src/main/java/com/hedvig/notificationService/customerio/hedvigfacades/MemberServiceImpl.kt

```diff
@@ -1,10 +1,15 @@
 package com.hedvig.notificationService.customerio.hedvigfacades
 
 import com.hedvig.notificationService.serviceIntegration.memberService.MemberServiceClient
+import com.hedvig.notificationService.serviceIntegration.memberService.dto.PersonHasSignedBeforeRequest
 import java.util.Locale
 
 class MemberServiceImpl(private val memberServiceClient: MemberServiceClient) {
     fun getPickedLocale(memberId: String): Locale {
         return Locale.forLanguageTag(memberServiceClient.pickedLocale(memberId).body.pickedLocale.replace('_', '-'))
     }
+
+    fun hasSignedBefore(request: PersonHasSignedBeforeRequest): Boolean {
```

Comment:
Maybe this is a bit nitty-gritty but I would go with
```suggestion
    fun hasPersonSignedBefore(request: HasPersonSignedBeforeRequest): Boolean {
```

Reply:
or
```suggestion
    fun hasPersonBeenSigned(request: HasPersonBeenSignedRequest): Boolean {
```